driver-core/build.gradle

```diff
@@ -64,20 +64,14 @@ afterEvaluate {
     jar.manifest.attributes['Automatic-Module-Name'] = 'org.mongodb.driver.core'
     jar.manifest.attributes['Bundle-SymbolicName'] = 'org.mongodb.driver-core'
     jar.manifest.attributes['Import-Package'] = [
-        'org.bson.*', // unfortunate that this is necessary, but if it's left out then it's not included
-        'javax.crypto.*',
-        'javax.management.*',
-        'javax.naming.*',
-        'javax.net.*',
-        'jdk.net.*',
-        'javax.security.sasl.*',
-        'javax.security.auth.callback.*',
-        'org.ietf.jgss.*',
-        'io.netty.*;resolution:=optional',
-        'org.xerial.snappy.*;resolution:=optional',
-        'com.github.luben.zstd.*;resolution:=optional',
-        'org.slf4j.*;resolution:=optional',
-        'jnr.unixsocket.*;resolution:=optional',
-        'com.mongodb.crypt.capi.*;resolution:=optional'
+            '!sun.misc.*',  // Used by DirectBufferDeallocator only for java 8
```

Comment:
Even given the comment, I'm not clear on why these need to be explicitly excluded.

Reply:
I removed them because they weren't in the original and due to the forking logic between jvm versions. Java 8 and Java 9+ work differently.